kitsune/sumo/utils.py

```diff
@@ -265,22 +265,22 @@ def is_ratelimited(request, name, rate, method="POST"):
     """
     if request.user.has_perm("sumo.bypass_ratelimit"):
         request.limited = False
-    else:
-        limited = is_ratelimited_core(
-            request, group=name, key="user_or_ip", rate=rate, method=method, increment=True
+        return False
+
+    if limited := is_ratelimited_core(
+        request, group=name, key="user_or_ip", rate=rate, method=method, increment=True
+    ):
+        # We only record a ratelimit event for this counter.
+        if request.user.is_authenticated:
+            key = f'user "{request.user.username}"'
```

Comment:
super nit: I would prefer double quotes externally and single internally just b/c we have all strings with double quotes because of black.